app/views/registrationamendment/section.scala.html

```diff
@@ -27,6 +27,7 @@
 @progressText = {
    @status match {
        case Started => {@Messages("progress.visuallyhidden.view.started")}
+       case NotStarted => {@Messages("progress.visuallyhidden.view.started")}
```

Comment:
Is 'progress.visuallyhidden.view.started' correct? Same as for 'Started'?

Reply:
yes, started is when you add/start a section and you didnt complete it